macros/src/lib.rs

```diff
@@ -11,12 +11,12 @@
 #![crate_type="dylib"]
 
 #![feature(plugin_registrar, quote)]
-#![feature(rustc_private, core, std_misc)]
+#![feature(rustc_private, convert)]
```

Comment:
I tried to change them. No difference. Besides, they are in the macro crate, which is built before the html5ever crate. I can still change it if it's desired.